subtests/docker_cli/run/run.py

```diff
@@ -46,7 +46,16 @@ def init_subargs(self):
         fqin = self.sub_stuff['fqin']
         subargs.append(fqin)
         subargs += get_as_list(self.config['bash_cmd'])
-        subargs.append(self.config['cmd'])
+        install_cmd = self.config['install_cmd'].strip()
```

Comment:
So, taking a step back, does `install_cmd` actually give any benefit? It seems to increase code complexity and break quoting (fixed here in `run.ini` but leaves existing `config_custom` files which will need to be edited). It looks to me like the approach started off worthwhile but didn't pan out. What would you think of reverting back to just simple `cmd`?